core/src/main/java/org/frankframework/configuration/AdapterManager.java

```diff
@@ -183,15 +184,24 @@ public void stop() {
 		List<Adapter> adapters = getAdapterList();
 		Collections.reverse(adapters);
 		for (Adapter adapter : adapters) {
-			log.info("stopping adapter [{}]", adapter::getName);
-			adapter.stopRunning();
+			stopAdapter(adapter);
 		}
 
 		updateState(RunState.STOPPED);
 	}
 
+	private void stopAdapter(Adapter adapter) {
+		log.info("stopping adapter [{}]", adapter::getName);
+		adapter.stopRunning();
+	}
+
+	/**
+	 * Closes this AdapterManager.
+	 * All adapters are removed from the Manager and you're unable to (re-)start after it's been closed!
+	 */
 	@Override
 	public void close() {
+		active.set(true);
```

Comment:
O god...
```suggestion
		active.set(false);
```

Reply:
In 8.0 en ouder zouden hier unit tests op stuk zijn gegaan maar ik geloof dat het nu wel zal werken. 🤞